inputfiles/overridingTypes.json

```diff
@@ -2400,6 +2433,274 @@
                         }
                     }
                 }
+            },
+            "ReadableStream": {
+                "name": "ReadableStream",
+                "override-exposed": "Window Worker",
```

Comment:
`name` and `override-exposed` are redundant here. Yes, `name`s in e.g. Worker are also redundant, but no need to fix them in this PR.